app/controllers/media_controller.rb

```diff
@@ -29,21 +29,25 @@ def stream
 
   private
 
+  def allowed_params
+    params.permit(:action, :id, :file_name, :format)
```

Comment:
Hmm, I don't quite understand it, but that's okay. The tests indicate that there is a need somewhere.  I doubt this will have any negative consequences, so I'm 👍 to keeping it in unless a reason presents itself.